src/OrbitGl/CaptureViewElementTester.cpp

```diff
@@ -8,10 +8,50 @@
 
 #include <unordered_map>
 
+orbit_gl::CaptureViewElementTester::CaptureViewElementTester()
```

Comment:
Why we are not in namespace orbit_gl?

Reply:
Good question. I'd like to do this in a follow-up PR though, because there's a bunch of PRs building on this one and this will be quite a pain to change now.

Reply:
Created b/230475928 to track